### PR TITLE
Mobile header styles

### PR DESCRIPTION
**Pull request by acacheung**
#100 
@Ctannerweb :eyes:

### CODE REVIEW

File: 2016/styles/modules/header.css [hunk 1]
```diff
@@ -1,54 +1,155 @@
...
+  @media (max-width: 939px) {
+    margin-right: auto;
+    margin-left: auto;
+  }
+}
+
+.nav__links {
+  @media (max-width: 939px) {
+    position: fixed;
+    top: 44px;
```
Comment (Ctannerweb) [inline 2016/styles/modules/header.css]: Change to top: 49px; to remove header height issue when opening and closing nav

File: 2016/styles/modules/header.css [hunk 2]
```diff
@@ -1,54 +1,155 @@
...
 
-.nav {
-  padding-top: 32px;
-  padding-bottom: 32px;
-  float: right;
+.club-sandwich {
+  @media (max-width: 939px) {
+    padding: 14px 5.625% 14px 14px;
+    float: right;
+    .mobile-nav-is-showing & {
```
Comment (Ctannerweb) [inline 2016/styles/modules/header.css]: Do we want nesting when we are looking for parent classes? So we only limit nesting when we want to add onto a classname.

File: 2016/styles/modules/header.css [hunk 3]
```diff
@@ -1,54 +1,155 @@
...
+    padding-top: calc(10vh - 20px);
+    display: block;
+    height: calc(20vh - 8.8px);
+    box-sizing: border-box;
+    font-size: 30px;
+    text-align: center;
+    transition-property: background-color;
     &:hover {
-      color: var(--white);
+      background-color: rgba(var(--white), .2);
```
Comment (Ctannerweb) [inline 2016/styles/modules/header.css]: Don't think the background-color on hover is needed / var(--white) inside the rgba wont work :)

File: 2016/styles/modules/header.css (hunk 2 above)
Comment (acacheung) [inline 2016/styles/modules/header.css]: we're also doing it in the theme.css file. i think im okay with this but do you have other suggestions

Comment (Ctannerweb) [inline 2016/styles/modules/header.css]: Nope that is fine with me just wanted to make sure :+1:

**Pull request merged**